article/app/views/fragments/articleBodyImmersive.scala.html

```diff
@@ -0,0 +1,68 @@
+
+@(model: ArticlePage, amp: Boolean = false)(implicit request: RequestHeader)
+
+@import common.LinkTo
+@import views.BodyCleaner
+@import views.support.TrailCssClasses.articleToneClass
+
+@defining(model.article) { article =>
+    <div class="l-side-margins">
+        <article id="article" data-test-id="article-root"
+            class="content content--article content--immersive tonal tonal--@articleToneClass(article) section-@article.sectionName.trim.toLowerCase.replaceAll("""[\s-]+""", "-")
+            @if(article.isAdvertisementFeature){content--advertisement-feature}
+            @if(article.isFeature && article.hasShowcaseMainElement){has-feature-showcase-element}"
+            itemscope itemtype="@article.schemaType" role="main">
+            <meta itemprop="mainEntityOfPage" content="@LinkTo(article.url)">
+            <div itemprop="publisher" itemscope itemtype="https://schema.org/Organization">
+                <meta itemprop="name" content="The Guardian">
+                @if(amp) {
```

Comment:
This was copied from the main `articleBody.scala.html` template. I wonder if this is just a remnant from that @stephanfowler / @NataliaLKB ?

If it is |'ll remove it from here and the main Body template.